RFCs/RFC-0: Lightweight RFC process.md

```diff
@@ -0,0 +1,69 @@
+
+All RFCs are submitted via PRs which will be merged when approved.
+
+### Collaboration
+Comments and feedback should be provided on the PR and focus on the technical content.
+
+Authors should address all comments written by the deadline. This doesn't mean every comment and suggestion must be accepted and incorporated, but they should be carefully read and responded to.
+
+Every RFC has a lifecycle. The life cycle has the following phases:
+* **Active**: The RFC has been drafted and is open for comments. When making the PR, the doc should start out in this state.
+* **Accepted**: All the comments have been addressed and the changes proposed on this RFC have been accepted. This should be the state while merging the PR.
+* **Dropped**: The changes proposed on this RFC aren't in effect anymore, the document is kept for historical purposes and there is a new RFC that’s more current.
+
+### Approval
+The proposal should be posted with a date by which the author would like to see the approval decision to be made. How much time is given to comment depends on the size and complexity of the proposed changes, but ideally should be from 2 days to a week.
+
+### When to write an RFC?
+There is always the option of going straight to a pull request. However, for larger changes, it might be wise to de-risk the risk of rejection of the pull request by first gathering input from the community.
+
+It’s encouraged to write an RFC for any major change. A major change might be:
+* Addition of any major new feature or subsystem
+* Changes that impact existing, or introduce new public APIs.
+* Changes that need to be coordinated across multiple repositories.
+
+### How to write an RFC?
+1. Copy the RFC template (in same folder as this document) and write your proposal! It's up to the author's discretion to decide which section in the template make sense for their proposal. It's recommended to cover the problem the proposal is solving, who it affects, how you’re proposing to solve it, and answers to frequently asked questions. Explicitly listing the goals will also make it easier to retrospect later on the proposal.
+2. Add your RFC to the riff/rfcs source directory, update to the next unique number. 
+3. Post a PR for your RFC prefixing the title with `RFC-#`, where `#` is the number of your RFC. Make sure to state the deadline clearly in your PR. 
+4. Answer questions and concerns on the PR. Consider adding questions that get asked more than once to the FAQ section of the RFC.
+5. After the deadline for feedback has been reached summarize the consensus and your decision on the PR  thread. 
+    1. If the decision is to drop the proposal, the status should be updated to Dropped. 
+    2. If we are moving forward the status goes to *Active*
```

Comment:
What's the status of the RFC prior to this?

Reply:
The status here should be Accepted. fixed.